src/gc/gc.cpp

```diff
@@ -11517,9 +11517,9 @@ void gc_heap::adjust_limit_clr (uint8_t* start, size_t limit_size, size_t size,
         if (gen_number == 0)
         {
             size_t pad_size = Align (min_obj_size, align_const);
-            make_unused_array (acontext->alloc_ptr, pad_size);
             dprintf (3, ("contigous ac: making min obj gap %Ix->%Ix(%Id)", 
                 acontext->alloc_ptr, (acontext->alloc_ptr + pad_size), pad_size));
+            make_unused_array (acontext->alloc_ptr, pad_size);
```

Comment:
> make_unused_array (acontext->alloc_ptr, pad_size); [](start = 11, length = 51)

is there any reason for this change?

Reply:
I used tracing for some investigations and having trace form the nested make_unused_array before the outer trace was confusing.

I basically moved the "contiguous ac: making gap ..."  thing before we call make_unused_array so that the reason why we do unused array would appear in the trace before the actual make_unused_array, not after.

---
In reply to: [287066315](https://github.com/dotnet/coreclr/pull/24504#discussion_r287066315) [](ancestors = 287066315)

Reply:
If that is not a general pattern used for tracing, I can move it back. It felt a bit easier to read.

---
In reply to: [287090700](https://github.com/dotnet/coreclr/pull/24504#discussion_r287090700) [](ancestors = 287090700,287066315)

Reply:
I see; makes sense. I'm fine with keeping the change.